pkgs/development/go-modules/generic/default.nix

```diff
@@ -130,7 +130,7 @@ let
         echo "$d" | grep -q "\(/_\|examples\|Godeps\)" && return 0
         [ -n "$excludedPackages" ] && echo "$d" | grep -q "$excludedPackages" && return 0
         local OUT
-        if ! OUT="$(go $cmd $buildFlags "''${buildFlagsArray[@]}" -v $d 2>&1)"; then
+        if ! OUT="$(go $cmd $buildFlags "''${buildFlagsArray[@]}" -v -p 1 $d 2>&1)"; then
```

Comment:
mhm. Would it be better to just remove the parallel xargs?

Reply:
@Mic92 - I pushed a commit that removes the parallel `xargs` in favor of passing `$NIX_BUILD_CORES` through to the compiler. Thoughts? If the new approach looks good, I can squash the commits together.